app/assets/stylesheets/application.scss

```diff
@@ -141,3 +153,27 @@ tr input[type=text] {
   padding-right: 10px;
   padding-left: 10px;
 }
+
+.bottom-border {
+  border-bottom: 1px solid #ccc;
+}
+
+.cancelled {
+  background-color: #ffebee;
+}
+
+.paid {
+  background-color: #e1f5fe;
+}
+
+.completed {
+  background-color: #e8f5e9;
+}
+
+.ordered {
+  background-color: #fff8e1;
```

Comment:
Color literals like `#fff8e1` should only be used in variable declarations; they should be referred to via variable everywhere else.